personal_assistant/pa_note/templates/pa_note/note_list.html

```diff
@@ -0,0 +1,19 @@
+{% extends "pa_core/home.html" %}
+
+{% block content %}
+    <h1>List of notes</h1>
```

Comment:
Використай існуючий блок для заголовка сторінки:
```suggestion
{% block title %}Notes{% endblock %}

{% block content %}
```